paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/model/CartState.kt

```diff
@@ -36,3 +39,16 @@ data class CartState(
         )
     }
 }
+
+@OptIn(ExperimentalPaymentSheetDecouplingApi::class)
```

Comment:
Is this supposed to be part of this change?

Reply:
No, it isn’t. Must have creeped in when I jumped between the various decoupling-related branches. I have removed it now.